src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -1001,6 +1013,7 @@ def __call__(
             return_dict=return_dict,
             deterministic=not train,
             rngs=rngs,
+            input_embeds = input_embeds
```

Comment:
(nit) Could you move this variable position to the last position for the `__call__` method (i.e. after `deterministic`)?